data_validation/config_manager.py

```diff
@@ -386,7 +390,7 @@ def get_source_ibis_table_from_query(self):
     def get_source_ibis_calculated_table(self, depth=None):
         """Return mutated IbisTable from source
         n: Int the depth of subquery requested"""
-        if self.validation_type == consts.CUSTOM_QUERY:
+        if self.source_table is None:  # either custom query or partition custom query
```

Comment:
Why do we have to change this? Won't generate-partitions custom query also set the validation_type to 'CUSTOM_QUERY'?

Reply:
Good catch. I had made this change before I decide to code the validation_type. Now that I made that change, this code can revert back to the more elegant way of doing things. Thank you.